models/users.py

```diff
@@ -43,10 +38,26 @@ def create_tables(cls) -> str:
             log.info("Created the users table")
             return "Created users table."
 
+    @classmethod
+    def get_user(cls, nick: str) -> Union["User", "AnonymousUser"]:
+        user: Union[User, AnonymousUser]
+        try:
+            if not isfile(db_path) or not cls.table_exists():
+                raise DatabaseError("Users db and table not created yet.")
+            user: User = cls.get(cls.nick == nick)
+        except User.DoesNotExist:
+            user: AnonymousUser = AnonymousUser()
+
+        return user
```

Comment:
This should be a utility function outside of the `User` class since it may return a non-`User` object.